### PR TITLE
Add retry for JWT auth to void application startup failed

### PR DESCRIPTION
**Pull request by baixinsui**
Fixed https://github.com/eclipse-xpanse/xpanse/issues/2178
Related to https://github.com/eclipse-xpanse/tofu-maker/pull/89
Related to https://github.com/eclipse-xpanse/terraform-boot/pull/171

When connect to IAM server failed will retry:
image

### CODE REVIEW

**Comment by Alice1319** [comment]
LGTM

**Comment by dengyali0125** [comment]
LGTM

**Comment by yuanxiaojun987** [comment]
LGTM

File: modules/security/pom.xml [hunk 1]
```diff
@@ -52,5 +52,10 @@
             <artifactId>wiremock-standalone</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (swaroopar) [inline modules/security/pom.xml:55]: can we remove this dependency if nimbus is not used anymore?

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: modules/security/pom.xml (hunk 1 above)
Comment (baixinsui) [reply modules/security/pom.xml:49]: Removed

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**